matlab/lpo_mex.cpp

```diff
@@ -328,51 +292,28 @@ std::shared_ptr<SeedFunction> createSeed( const std::string & s ) {
     }
     return seed;
 }
+static std::string model;
```

Comment:
Do we need a static model variable? It would be nicer to load the model directly in the newProposal function and then just keep it around.